Source/cmake/OptionsCommon.cmake

```
@@ -159,6 +159,12 @@ if (CXX_COMPILER_SUPPORTS_GSPLIT_DWARF AND LD_SUPPORTS_SPLIT_DEBUG)
   set(ENABLE_DEBUG_FISSION_DEFAULT ON)
 endif ()
 
+if (ENABLE_DEBUG_FISSION_DEFAULT AND SCCACHE_FOUND AND COMPILER_IS_CLANG AND NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
```

After some discussion, we decided to approve this if you test for RelWithDebInfo too:

```suggestion
if (ENABLE_DEBUG_FISSION_DEFAULT AND SCCACHE_FOUND AND COMPILER_IS_CLANG AND NOT CMAKE_BUILD_TYPE STREQUAL "Debug" AND NOT CMAKE_BUILD_TYPE STREQUAL "RelWithDebInfo")
```

---

RelWithDebInfo GTK clang sccache build passed. Thanks for the review!